crypto/aes/aes_ige.c

```diff
@@ -36,17 +36,30 @@ typedef struct {
 
 void AES_ige_encrypt(const unsigned char *in, unsigned char *out,
                      size_t length, const AES_KEY *key,
-                     unsigned char *ivec, const int enc)
+                     unsigned char *ivec, const int enc) {
+    /* Ignore the return value since we are void */
+    AES_ige_encrypt_ex(in, out, length, key, ivec, enc);
```

Comment:
Well technically adding an assert is not right because, if it failed, it is because the caller supplied incorrect parameters. However in this case perhaps it is the right thing to do....I'm not really sure.